configs/cifar10_vkd.yaml

```diff
@@ -3,6 +3,7 @@ epochs: 100
 output_dir: save
 resume: null
 save_interval: 10
+load_model_path: models/teacher.pt
```

Comment:
where does this come from?

Reply:
This suppose to be the location of the teacher model